aten/src/ATen/native/native_functions.yaml

```diff
@@ -12816,6 +12816,12 @@
   tags: view_copy
   autogen: _test_autograd_multiple_dispatch_view_copy.out
 
+# Note: this function is only for testing.
+- func: _test_inductor_realize(Tensor self) -> Tensor
```

Comment:
I think we shouldn't be adding testing ops into pytorch proper, we can do python registration directly in the test instead, cc @SherlockNoMad

Reply:
Is it possible to register the backward/forward formulas from python?

Reply:
Forward/backward can be handled with AutogradFunction, I don't remember though if we trace through it (which would work) or treat it as a leaf module (that won't).